.ci/docker/docker-compose.yml

```diff
@@ -143,45 +143,30 @@ services:
     volumes:
       - nodelocalstackdata:/var/lib/localstack
 
-  zookeeper:
-    # https://hub.docker.com/r/bitnami/zookeeper/tags
-    image: bitnami/zookeeper:3.9.1
-    ports:
-      - "2181:2181"
-    volumes:
-      - nodezookeeperdata:/bitnami
-    environment:
-      - ALLOW_ANONYMOUS_LOGIN=yes
-    healthcheck:
-      # Using 'srvr' instead of the more common 'ruok' because this bitnami
-      # image does not have 'ruok' on the '4lw.commands.whitelist' in zoo.cfg.
-      test: ["CMD-SHELL", "echo srvr | nc -w 2 localhost 2181"]
-      interval: 5s
-      timeout: 5s
-      retries: 5
-
   kafka:
-    # https://hub.docker.com/r/bitnami/kafka/tags
-    image: bitnami/kafka:3.3.2
+    image: apache/kafka:4.1.0
     ports:
-      - "9093:9093"
+      - "9094:9094"
     volumes:
       - nodekafkadata:/var/lib/kafka/data
     environment:
-      - KAFKA_BROKER_ID=1
-      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
-      - ALLOW_PLAINTEXT_LISTENER=yes
-      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CLIENT:PLAINTEXT,EXTERNAL:PLAINTEXT
-      - KAFKA_CFG_LISTENERS=CLIENT://:9092,EXTERNAL://:9093
-      - KAFKA_CFG_ADVERTISED_LISTENERS=CLIENT://kafka:9092,EXTERNAL://kafka:9093
-      - KAFKA_CFG_INTER_BROKER_LISTENER_NAME=CLIENT
-      - KAFKA_CFG_DELETE_TOPIC_ENABLE=true
-    depends_on:
-      - zookeeper
```

Comment:
The apache/kafka image doesn't depend on having a zookeeper? Or do you know if it embeds its own ZK?